### ISSUE TITLE
[Tooltip] 点击trigger后再点击弹窗，弹窗会隐藏

### ISSUE THREAD

**Issue by YannLynn** [DouyinFE/semi-design#977]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Component

Tooltip

### Semi Version

latest

### Current Behavior

1. 点击trigger
2. 点击弹层
3. 弹层被关闭

### Expected Behavior

1. 点击trigger
2. 点击弹层
3. 弹层不被关闭

### Steps To Reproduce

1. 点击trigger
2. 点击弹层
3. 弹层被关闭

https://user-images.githubusercontent.com/101160769/179733097-77564490-5f49-47d5-bf4d-b680a3767a64.mp4

### ReproducibleCode

```markdown
import React from 'react';
import { Tooltip, Tag } from '@douyinfe/semi-ui';

function Demo() {
    return (
        <Tooltip content={'hi bytedance'}>
            <Tag>悬停此处</Tag>
        </Tooltip>
    );
}
```

### Environment

```markdown
- OS:
- browser:
```

**Comment by pointhalo** [comment]
这个是v2.15引入的问题？

**Comment by pointhalo** [comment]
fix 2.17.0-beta.0
